instruqt-tracks/nomad-multi-server-cluster/track.yml

```diff
@@ -161,7 +161,7 @@ challenges:
     ```
     You should see a single line like `[1] 3194`, which gives the PID of the Consul server.
 
-    Now,look at the `consul-server2.json` configuration file on the "Config Files" tab on the left. This file is very similar to `consul-server1.json` but will make the the second Consul server repeatedly try to join to the Consul servers running on the `nomad-server-1` and `nomad-server-3` VMs.
+    Now look at the `consul-server2.json` configuration file on the "Config Files" tab on the left. This file is very similar to `consul-server1.json` but will make the the second Consul server repeatedly try to join to the Consul servers running on the `nomad-server-1` and `nomad-server-3` VMs.
```

Comment:
I think use of comma was correct. See https://linguaholic.com/linguablog/comma-before-or-after-now/#:~:text=comma%20after%20%E2%80%9Cnow%E2%80%9D-,When%20%E2%80%9Cnow%E2%80%9D%20is%20used%20as%20an%20introductory%20word,by%20a%20comma%20right%20after.

Reply:
I'm not passionate either way, so if it's extra work to roll those back I apologize.